lib/Page.js

```diff
@@ -775,20 +775,16 @@ class Page extends EventEmitter {
    * @param {!Array<string>} values
    */
   async select(selector, ...values) {
-    await this.$eval(selector, (element, values) => {
+    return await this.$eval(selector, (element, values) => {
```

Comment:
nit: can you please update jsdoc on line 775? it should have the "return" statement

Reply:
Like this?
 `- returns: <[Promise]<[Array]<[string]>>> Returns an array of option values that have been successfully selected.`

I put that in there but its on line 928. Line 775 in my api.md file refers to Page.goForward().

Reply:
I mean the jsdoc for this function (that block comment with @param tags inside)

Reply:
oh i see

Reply:
where can i find documentation about the !, ? and | modifiers for jsdoc? I dont know what they do.

Reply:
@AlexChung1995 indeed. Briefly speaking, the "!" is non-nullable type, the "?" is a nullable type. More info could be found here: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler